ui/shared/components/form/FormWizard.jsx

```diff
@@ -30,31 +31,39 @@ class FormWizard extends React.PureComponent {
 
   resolvedPageSubmit = () => Promise.resolve()
 
+  setSubmitSucceeded = () => this.setState({ formSubmitSucceeded: true })
+
   onFormSubmit = (values) => {
     const { onSubmit } = this.props
     const { asyncValues } = this.state
     return onSubmit({ ...asyncValues, ...values })
   }
 
   render() {
-    const { pages, onSubmit, ...props } = this.props
-    const { pageIndex } = this.state
+    const { pages, onSubmit, successMessage, ...props } = this.props
+    const { pageIndex, formSubmitSucceeded } = this.state
 
     const { fields, onPageSubmit } = pages[pageIndex]
 
-    const formProps = pageIndex === pages.length - 1 ? { onSubmit: this.onFormSubmit } : {
+    const formProps = (pageIndex === pages.length - 1) ? {
+      onSubmit: this.onFormSubmit,
+      onSubmitSucceeded: this.setSubmitSucceeded,
+    } : {
       onSubmit: onPageSubmit(this.onPageSubmitSucceeded) || this.resolvedPageSubmit,
       onSubmitSucceeded: this.navigateNext,
       submitButtonText: 'Next',
       submitButtonIcon: 'angle double right',
     }
+
     const backButtonProps = pageIndex === 0 ? {} : {
       onCancel: this.navigateBack,
       cancelButtonText: 'Back',
       cancelButtonIcon: 'angle double left',
     }
 
-    return (
+    return (formSubmitSucceeded ? (
+      <b>{successMessage}</b>
```

Comment:
This I think needs better formatting. I think using a Header would be good to get some padding and also a success icon - let me know what you think of this:
```
<Header
    icon="check circle"
    content="Request Submitted"
    subheader={successMessage}
  />
```